index.js

```diff
@@ -1,296 +1,44 @@
 'use strict';
 
+var types = require('./validator/cardTypes.json');
```

Comment:
can we just make this folder `lib` instead of `validator`?

Reply:
Renamed to lib